### ISSUE TITLE
MCHAD support

### ISSUE THREAD

**Issue by r2dev2** [luna-translations-bot/luna-translations-bot#6]
In heavy youtube chats, many translations from translators get deleted. Some translators may also be shadow banned and have all lines greater than 10 characters deleted. To solve this, many have migrated over to using mchad, an app that sends translations to both youtube chat and a separate API. It would be nice if this bot could also support using mchad.

Here is how an app I work on implemented support:
https://github.com/LiveTL/LiveTL/blob/525ae4a9c8462d3338c02232ccdbef838a3b17c6/src/js/mchad.js#L179

[MCHAD API docs](http://mchatx.org/docs#line9)

**Comment by tam-carre** [comment]
Yeah, that's planned for when I have the time. Thanks for the LiveTL example
